raml/APIs/NmosOauthAPI.raml

```diff
@@ -0,0 +1,152 @@
+#%RAML 1.0
+
+# AMWA NMOS BCP-003 Security: NMOS OAuth API with JSON Web Tokens
+# (c) AMWA 2019
+
+title: NMOS Oauth
+baseUri: http://example.api.com/x-nmos/oauth/{version}
+version: v1.0
+mediaType: application/json
+securitySchemes: !include SecuritySchemes.raml
+
+documentation:
+  - title: Overview
+    content: |
+      The NMOS Oauth Security API is exposed by an NMOS Authorization Server as a standard interface for requesting, revoking, and managing tokens and client registration
+  - title: Endpoint Discovery
+    content: |
+      The NMOS OAuth Authorization Server URL is discoverable using unicast and/or multicast DNS using the '_nmos-security._tcp' service name
```

Comment:
That's fair enough, despite the slightly upsetting 'z'

Reply:
Ha ha, not just me then. 😂

Reply:
Interestingly I've used "s" in the rest of the spec. Oh well :-/

Reply:
My fingers did object to having to use the Z key, but -ize -ization etc. would be consistent with other AMWA specs/docs.  Similarly with "cipher" not "cypher" which we did before.

Reply:
Most SDOs standardize on US english (sorry!)  But I'm also told that the The Oxford English Dictionary, which usually favors British spellings, still lists "authorize" as the primary spelling.

Reply:
Note we have to be a bit careful on length "The Service Name may be up to 15 bytes, plus the underscore and length byte, making a total of 17. Including the "_udp" or "_tcp" and its length byte, this makes 22 bytes." RFC 6763

Reply:
I feel we can kill two birds with stone by using _nmos-auth._tcp. Short enough and isn't partial on spelling...

In general the IETF specifies that RFCs can be in UK or US spelling as long as documents are self consistent. Given that we're dealing with headers that use the US spelling we should probably use that to be self consistent. However - I will fix that once I've finished the spec, because my muscle memory will only keep putting more 's'es in so I may as well.

Now, if only I'd gotten as much interest and debate on the rest of the specification...